test/azure/version-tolerant/Expected/AcceptanceTests/CustomUrlPagingVersionTolerant/custombaseurlpagingversiontolerant/aio/__init__.py

```diff
@@ -8,10 +8,16 @@
 
 from ._auto_rest_parameterized_host_test_paging_client import AutoRestParameterizedHostTestPagingClient
 
+try:
+    from ._patch import __all__ as _patch_all
+    from ._patch import *  # pylint: disable=unused-wildcard-import
+except ImportError:
+    _patch_all = []
+from ._patch import patch_sdk as _patch_sdk
+
 __all__ = ["AutoRestParameterizedHostTestPagingClient"]
+__all__.extend([p for p in _patch_all if p not in __all__])
 
 # `._patch.py` is used for handwritten extensions to the generated code
 # Example: https://github.com/Azure/azure-sdk-for-python/blob/main/doc/dev/customize_code/how-to-patch-sdk-code.md
-from ._patch import patch_sdk
-
-patch_sdk()
+_patch_sdk()
```

Comment:
I think it is better to move the annotation to class annotation of `def patch_sdk():`, which is more convenient for customers to find the reference link